README.md

```diff
@@ -75,3 +75,43 @@ trainer = Trainer(agent, env, mode="dyna", model="tabular", n_episodes=10000)
 episode_rewards = trainer.train()
 trainer.plot(episode_rewards)
 ```
+
+## Algorithms
+
+### Deep RL
+ - DQN (Deep Q Networks)
+    - DQN
+    - Double DQN
+    - Dueling DQN
+    - Noisy DQN
+    - Categorical DQN
```

Comment:
Prioritized Replay DQN too

Reply:
Can all of them work with it?

Reply:
Not yet. None of them can work with each other atm. There's an issue for this #209

Reply:
So lets leave it then. Prioritized Replay is more of a utility. We might need to get rid of it completely after Reverb Intergration #233 #151